app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,198 @@
+      image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+      image_2_path: "static/content/hero-images/0007.jpg",
+      image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
+    ) do %>
+       <p>
+    As a physics teacher, you’ll be engaging students in a wide range of experiments that can spark their interest.
+  </p>
+
+  <p>
+    You can inspire your students to consider rewarding careers from tackling climate change to artificial intelligence.
+  </p>
+    <% end %>
+</section>
+<section class="col col-720">
+      <h2>What you'll be teaching</h2>
+      <p>
+      You'll teach the content of the national curriculum, with opportunities to develop your students’ ability to think like physicists and see physics as a solution to many challenges. The themes you are likely to cover:
+      </p>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+      <div>
+        <p>
+          11 to 14 year olds (key stage 3):
+        </p>
+      </div>
+
+      <div>
+        <ul class="vertical-tags">
+          <li class="tag">Energy</li>
+          <li class="tag">Motion and forces</li>
+          <li class="tag">Electricity and electromagnetism</li>
+          <li class="tag">Waves and matter</li>
+        </ul>
+      </div>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <p>
+        14 to 16 year olds (key stage 4):
+      </p>
+    </div>
+
+    <div>
+       <ul class="vertical-tags">
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+    </div>
+ </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <p>
+        16 to 18 year olds (key stage 5):
+      </p>
+    </div>
+
+    <div>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+      </ul>
+    </div>
+ </section>
+
+  <section class="col col-720">
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search the internet for past exam papers to see the type of questions that have been covered.</p>
+
+    <p>
+      You may also need to teach other subjects, particularly for 11- to 16-year-olds and in some cases for post-16. This will vary depending on the individual schools.
```

Comment:
Different formatting here re `11- to 16-year-olds` - needs to be consistent throughout and in line with our style guide